Fix grammar errors (#718)
jaegertracing/documentation: content/docs/next-release/deployment.md

### Memory

The in-memory storage is not intended for production workloads. It's intended as a simple solution to get started quickly and

> data will be lost once the process is over.

Reviewer: ```suggestion
data will be lost once the process exits.
```

## Aggregation Jobs for Service Dependencies

> Production deployments need an external process that aggregates data and creates dependency links between services. Project [spark-dependencies](https://github.com/jaegertracing/spark-dependencies) is a Spark job which derives dependency links and stores them directly in the storage.

Reviewer: ```suggestion
Production deployments need an external process that aggregates data and creates dependency links between services. Project [spark-dependencies](https://github.com/jaegertracing/spark-dependencies) is a Spark job which derives dependency links and writes them directly to the storage.
```
